App/Redux/AuthRedux.ts

```diff
@@ -117,7 +118,7 @@ export function reducer (state: AuthState = initialState, action: AuthAction): A
 }
 
 export const AuthSelectors = {
-  tokens: (state: any) => state.auth.tokens
+  tokens: (state: RootState) => state.auth.tokens
```

Comment:
Can use `RootState` now wherever you have to handle state, nice and typed

Reply:
nice